include/polyscope/surface_mesh.h

```diff
@@ -51,46 +52,47 @@ class SurfaceGraphQuantity;
 
 template <> // Specialize the quantity type
 struct QuantityTypeHelper<SurfaceMesh> {
-  typedef SurfaceMeshQuantity type;
+    typedef SurfaceMeshQuantity type;
 };
 
 
 // === The grand surface mesh class
 
 class SurfaceMesh : public QuantityStructure<SurfaceMesh> {
-public:
-  typedef SurfaceMeshQuantity QuantityType;
+  public:
```

Comment:
It looks like a code formatter was run on the code (with different settings than the codebase's usual formatter). Unfortunately this creates spurious changes like this on many, which make it hard to resolve merges and track changes in the future.

Could you try reformatting the code with the `.clang-format` settings in the repo root?